package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "lint": "eslint src",
     "start": "run-p start:**",
-    "start:hugo": "hugo -d ../dist -s site -vw",
+    "start:hugo": "hugo -d ../dist -s site -vw --baseURL=/",
```

Comment:
http://localhost:<port>?

Reply:
I don't think it matters in development but fixd so it's same behavior